lib/cli/icgc-download-client.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import logging
+
+import yaml
+
+from clients.ega import ega_script
+from clients.gdc import gdc_script
+from clients.gnos import genetorrent
+from clients.icgc import icgc_script
+
+logger = logging.getLogger('__log__')
+logger.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--config", nargs='?', default="/icgc/mnt/configuration/config.yaml",
+                    help="File used to set download preferences and authentication criteria")
+parser.add_argument('repo', choices=['ega', 'collab', 'cghub', 'aws', 'gdc'],
+                    help='specify which repository to download from, all lowercase letters')
+parser.add_argument('file_id', help='Lowercase identifier of file or path to manifest file')
+parser.add_argument('-m', '--manifest', action='store_true', help='Flag used when the downloading from a manifest file')
+
+args = parser.parse_args()
+configtext = open(args.config, 'r')
```

Comment:
If `args.config` contains a bogus path that points to a file that doesn't exist this will throw an `IOError` 

This would be preferable:

```python
try:
    configtext = open(args.config, 'r')
except IOError:
    # Add a human readable log message here. 
```